### ISSUE TITLE
Training custom dataset wtih Python

### ISSUE THREAD

**Issue by mazik1496** [ultralytics/ultralytics#1352]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

It trains the dataset through roboflow and learns with Python code, but even though the epoch is set to 100, it only trains 3 times. I don't know what causes this.

### Minimal Reproducible Example

#!/usr/bin/env python3

from ultralytics import YOLO

# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

# Use the model
model.train(data='data.yaml', epochs=100)  # train the model
metrics = model.val()  # evaluate model performance on the validation set
results = model("/home/user/vrx_ws/src/vrx/vrx_gazebo/nodes/Result_of_Original.png")  # predict on an image
success = model.export(format="onnx")  # export the model to ONNX format

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by mazik1496** [comment]
@glenn-jocher I used this code and result.

-----source code-------
#!/usr/bin/env python3

from ultralytics import YOLO

# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

# Use the model
model.train(data='data.yaml', epochs=100)  # train the model
metrics = model.val()  # evaluate model performance on the validation set
results = model("/home/user/vrx_ws/src/vrx/vrx_gazebo/nodes/Result_of_Original.png")  # predict on an image
success = model.export(format="onnx")  # export the model to ONNX format
--------------------------------------------


![스크린샷, 2023-03-10 10-44-24](https://user-images.githubusercontent.com/80936392/224201875-bb19b735-719c-42b2-85ab-bb80b6c75b43.png)

**Comment by Laughing-q** [comment]
@mazik1496 I'm not able to reproduce the issue, `epochs` works properly to me.
![image](https://user-images.githubusercontent.com/61612323/224266200-8113d1c8-31ab-4722-ba54-060261941bf5.png)
